testingt.go

```diff
@@ -17,3 +20,13 @@ type TestingT interface {
 	Skipf(format string, args ...interface{})
 	Helper()
 }
+
+// rapidT is a wrapper around `*rapid.T` that stubs missing `TestingT`
```

Comment:
Have you considered/looked into sending a PR to `rapid.T` to add Cleanup in the root source?

Reply:
I actually had't, thanks for the suggestion. I was doing my best at the time to stop "going deeper". :sweat_smile: